quill-core/src/main/scala/io/getquill/norm/ApplyMap.scala

```diff
@@ -17,6 +17,7 @@ object ApplyMap {
       case Take(Map(a: GroupBy, b, c), d)         => None
       case Drop(Map(a: GroupBy, b, c), d)         => None
       case Map(a: GroupBy, b, c) if (b == c)      => None
+      case Map(a: Nested, b, c) if (b == c)       => None
```

Comment:
I don't want identity of `map(x => x.i).nested.map(x => x)` to be skipped, since I don't want to make guesses in `SqlQuery.scala:56`